scripts/githooks/pre-push-prevent-upstream-push

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# pre-push hook blocking push to the upstream brach.
+# You can disable this hook by calling push with --no-verify argument.
+#
+# If you want to use this hook then call:
+#
+# cp ./pre-push-prevent-upstream-push ../../.git/hooks/pre-push
+#
+# This hook is called with the following parameters:
+#
+# $1 -- Name of the remote to which the push is being done
+# $2 -- URL to which the push is being done
+#
+
+remote="$1"
+url="$2"
+
+prevent_remote=$(git remote -v | grep rhinstaller | head -n 1 | cut -f 1)
```

Comment:
There should be probably quotation marks: `prevent_remote="$(...)"`

Reply:
it works even without them so they are not technically required but yeah, it won't hurt to add them